#!/usr/bin/env python
import math

PACKAGE = "egocylindrical"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("inflation_radius", double_t, 0, "how much to inflate image", 0.2, 0)
gen.add("inflation_height", double_t, 0, "how much to inflate image", 0.2, 0)
gen.add("num_threads", int_t, 0, "# threads for inflating range image", 1, 1, 16)

exit(gen.generate(PACKAGE, "egocylindrical", "RangeImageInflatorGenerator"))
